src/Guest.js

```diff
@@ -1,13 +1,14 @@
 class Guest {
-  constructor(guessData) {
-    this.id = guessData.id;
-    this.name = guessData.name;
-    this.username = guessData.userName;
+  constructor(guestData) {
+    this.id = guestData.id;
+    this.name = guestData.name;
+    this.username = guestData.userName;
     this.password = 'overlook2020'
     this.bookings = [];
     this.totalSpent = 0;
   }
 
+
   addBooking(booking) {
     this.bookings.push(booking)
```

Comment:
This is good for adding the booking to the array. Are you going to set it up to POST to the web API using fetch?